src/Samples/Basic/Temporal.Samples.DesignProcess/Part4_2_BasicWorkflowUsage_MultipleWorkers.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+using System.Threading.Tasks;
+
+using Microsoft.Extensions.Hosting;
+
+using Temporal.Common.DataModel;
+using Temporal.Worker.Activities;
+using Temporal.Worker.Hosting;
+using Temporal.Worker.Workflows;
+using Temporal.Worker.Workflows.Base;
+
+namespace Temporal.Sdk.BasicSamples
+{
+    /// <summary>
+    /// Some very simple workflows. The level of abstraction is the same as in <see cref="Part4_1_BasicWorkflowUsage" />.
+    /// Here, we focus on multiple workers per process.    
+    /// </summary>
+    public class Part4_2_BasicWorkflowUsage_MultipleWorkers
+    {
+        public class SayHelloWorkflow : BasicWorkflowBase
+        {            
+            public override async Task<PayloadsCollection> RunAsync(WorkflowContext workflowCtx)
+            {
+                PayloadsCollection input = workflowCtx.CurrentRun.Input;
+                string addresseeName = workflowCtx.GetSerializer(input).Deserialize<string>(input);
+
+                string greeting = $"Hello, {addresseeName ?? "<null>"}.";
+                await SpeakGreetingAsync(greeting, workflowCtx);
+
+                return PayloadsCollection.Empty;
+            }
+
+            private async Task SpeakGreetingAsync(string text, WorkflowContext workflowCtx)
+            {
+                PayloadsCollection greetingPayload = workflowCtx.WorkflowImplementationConfig.DefaultPayloadSerializer.Serialize(text);
+                await workflowCtx.Orchestrator.Activities.ExecuteAsync("SpeakAGreeting1", greetingPayload);
+            }
```

Comment:
This is a bit the same as my other comment - users shouldn't need to explicitly serialize inputs before calling activities